ampersand-select-view.js

```diff
@@ -119,6 +121,7 @@ module.exports = View.extend({
         if (!this.select) throw new Error('no select found in template');
         if (matches(this.el, 'select')) this.select = this.el;
         if (this.select) this.select.setAttribute('name', this.name);
+        if (this.select) this.select.setAttribute('tabindex', this.tabindex);
```

Comment:
Help for other reviewers: this is one of the two actual additions that isn't whitespace cleanup.